app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { class: 'form' }) do |f| %>
+
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", class: 'text-area title-area' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio, autofocus: true, autocomplete: "bio", class: 'text-area title-area' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'text-area title-area' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", class: 'text-area title-area' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: 'text-area title-area' %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up", class: 'see-all btn pad-up' %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] When I successfully sign up with new credentials, I notice this error when I am redirected to the `Users#Index` page:

![image](https://github.com/Uthmanbello/myblog/assets/77106962/5f355380-8100-4f0e-b4a3-19a41a588ec8)

Upon investigation, I noticed that the way you referenced the `User` model in the **_Create Post_** migration file is incorrect and it causes this error because every instance of the `Post` model won't have the `author_id` attribute. You would need to fix this by referencing the `User` model properly. 😎 

```ruby
t.references :author, null: false, foreign_key: { to_table: :users }
```
After referencing the `User` model properly in the **_Create Post_** migration file, kindly delete this migration file:
`20230505134353_remove_user_id_from_posts.rb` so that there won't be problems when running migrations. Kindly implement this project. 👍 😎

Reply:
@Gambit142 ,

Thanks a lot

Reply:
@Uthmanbello, if you check the logs, you will see the link attached. When you click the link, it will show a flash message telling you that the email has been confirmed. Another alternative is to use sendgrid. You can read this article to understand that https://dev.to/setobiralo/using-devise-and-sendgrid-to-send-confirmation-email-on-rails-app-3h14 👍

Reply:
@Gambit142 ,

Thanks a lot for the reply. I have it here:
![screenshot email confirmation](https://github.com/Uthmanbello/myblog/assets/112436398/f91d9560-56ed-414c-b06b-db60f984cbc9)

Reply:
This change has been done well